config/github.php

```diff
@@ -78,6 +78,16 @@
             // 'enterprise' => false,
         ],
 
+        'private' => [
+            'method'     => 'file',
```

Comment:
This config option won't work unless the method id set to `private` surely?

Reply:
Fixed